static/app/components/events/autofix/autofixMessageBox.tsx

```diff
@@ -0,0 +1,261 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {Button} from 'sentry/components/button';
+import {type AutofixStep, AutofixStepType} from 'sentry/components/events/autofix/types';
+import Input from 'sentry/components/input';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {
+  IconCheckmark,
+  IconClose,
+  IconCode,
+  IconFatal,
+  IconQuestion,
+  IconSad,
+} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import marked, {singleLineRenderer} from 'sentry/utils/marked';
+import {useMutation} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+
+function useSendMessage({groupId, runId}: {groupId: string; runId: string}) {
+  const api = useApi();
```

Comment:
persist request on unmount
```suggestion
  const api = useApi({persistInFlight: true});
```